scripts/crawl_zenodo.py

```diff
@@ -548,17 +561,26 @@ def download_file(bucket, d, dataset_dir):
         if bucket["type"] == "zip":
             d.download_url(link, archive=True if bucket["type"] == "zip" else False)
         else:
-            annex("addurl", link, "--fast")
+            try:  # Try to addurl twice as rarely it might not work on the first try
```

Comment:
This is kind of hack-y.  What happens if there's a `GitCommandError` on the second try?  Just let it fail with an uncaught exception?

Reply:
Some explanation of the investigation about *why* it may be failing on first try but not on second try would help to motivate this hack.